apps/angular/di/src/app/app.component.ts

```diff
@@ -24,7 +25,14 @@ export class ProductDirective {
 
 @Component({
   standalone: true,
-  imports: [TableComponent, CurrencyPipe, AsyncPipe, NgFor, ProductDirective],
+  imports: [
+    TableComponent,
+    CurrencyPipe,
+    AsyncPipe,
+    NgFor,
+    ProductDirective,
+    CurrencyCodeDirective,
+  ],
   providers: [CurrencyService],
```

Comment:
On pourrait enlever le service des providers de AppComponent, non ?

Reply:
that's true 👍